### PR TITLE
Make frontend accept debts and transactions

### PR DESCRIPTION
**Pull request by thomaseizinger**
The backend will introduce a breaking change by renaming transactions to
debts. This change prepares the frontend for handling both until we can
be sure that our backend does no longer return us a `transactions`
property.

### CODE REVIEW

File: src/routes/App/modules/currentNobt/reducer.test.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      data: {
+        transactions: [
+          {},
+          {}
+        ]
+      }
+    }));
+
+
+    expect(newState.data.transactions).toEqual([{}, {}])
```
Comment (duffleit) [inline src/routes/App/modules/currentNobt/reducer.test.js]: newState.data.debts

Comment (thomaseizinger) [reply src/routes/App/modules/currentNobt/reducer.test.js]: Haha, what a fail :D
Will do :)

Comment (thomaseizinger) [reply src/routes/App/modules/currentNobt/reducer.test.js]: No wait, the code is correct. I did not want to introduce debts already. The response handling should convert debts to transactions.

Comment (thomaseizinger) [reply src/routes/App/modules/currentNobt/reducer.test.js]: The error was related to the linter because of a console statement.

**Pull request merged**